<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>EditCategory</class>
	<widget class="QWidget" name="EditCategory">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>520</width>
				<height>398</height>
			</rect>
		</property>
		<property name="sizePolicy">
			<sizepolicy hsizetype="Preferred" vsizetype="Preferred">
				<horstretch>0</horstretch>
				<verstretch>0</verstretch>
			</sizepolicy>
		</property>
		<layout class="QGridLayout" name="gridLayout">
			<item row="0" column="0">
				<widget class="QListWidget" name="listWidgetCategory"/>
			</item>
			<item row="0" column="1">
				<layout class="QVBoxLayout" name="verticalLayout_2">
					<property name="sizeConstraint">
						<enum>QLayout::SetMinimumSize</enum>
					</property>
					<item>
						<layout class="QVBoxLayout" name="verticalLayout">
							<item>
								<widget class="QLabel" name="labelNewName">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Fixed" vsizetype="Preferred">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="text">
										<string>New Name</string>
									</property>
									<property name="alignment">
										<set>Qt::AlignCenter</set>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QLineEdit" name="lineEditNewName">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Fixed" vsizetype="Fixed">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
								</widget>
							</item>
						</layout>
					</item>
					<item>
						<spacer name="verticalSpacer">
							<property name="orientation">
								<enum>Qt::Vertical</enum>
							</property>
							<property name="sizeHint" stdset="0">
								<size>
									<width>20</width>
									<height>40</height>
								</size>
							</property>
						</spacer>
					</item>
					<item>
						<widget class="QPushButton" name="buttonOK">
							<property name="sizePolicy">
								<sizepolicy hsizetype="Fixed" vsizetype="Fixed">
									<horstretch>0</horstretch>
									<verstretch>0</verstretch>
								</sizepolicy>
							</property>
							<property name="layoutDirection">
								<enum>Qt::RightToLeft</enum>
							</property>
							<property name="text">
								<string>OK</string>
							</property>
						</widget>
					</item>
					<item>
						<widget class="QPushButton" name="buttonExit">
							<property name="sizePolicy">
								<sizepolicy hsizetype="Fixed" vsizetype="Fixed">
									<horstretch>0</horstretch>
									<verstretch>0</verstretch>
								</sizepolicy>
							</property>
							<property name="layoutDirection">
								<enum>Qt::RightToLeft</enum>
							</property>
							<property name="text">
								<string>Exit</string>
							</property>
						</widget>
					</item>
				</layout>
			</item>
		</layout>
	</widget>
	<resources/>
	<connections/>
</ui>
